cmd/boulder-ra/main.go

```diff
@@ -162,7 +169,16 @@ func main() {
 		pubc = bgrpc.NewPublisherClientWrapper(pubPB.NewPublisherClient(conn))
 
 		if c.RA.CTLogGroups != nil {
-			ctp = ctpolicy.New(pubc, c.RA.CTLogGroups, logger)
+			groups := make([]cmd.CTGroup, len(c.RA.CTLogGroups))
+			for i, logs := range c.RA.CTLogGroups {
+				groups[i] = cmd.CTGroup{
+					Name: fmt.Sprintf("%d", i),
```

Comment:
In `test/config-next/ra.json` you specifically set a `name` key in each `CTLogGroups2` entry, but here the name is always derived from loop index. I think that's a bug?

Reply:
This is just a stop gap until we can remove the deprecated `CTLogGroups` config option which is just slice of slices so doesn't have a name field.

Reply:
Ahhh, I understand. Makes sense :+1: